datastore/db/bytebase/FertiScan/schema_0.0.9.sql

```diff
@@ -0,0 +1,252 @@
+    "upload_date" date NOT NULL DEFAULT current_timestamp,
+    "name" text
+    );
+        
+    alter table "fertiscan_0.0.9".users ADD "default_set_id" uuid REFERENCES "fertiscan_0.0.9".picture_set(id);
+
+    CREATE TABLE "fertiscan_0.0.9"."picture" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture" json NOT NULL,
+    "nb_obj" int,
+    "picture_set_id" uuid NOT NULL REFERENCES "fertiscan_0.0.9".picture_set(id),
+    "verified" boolean NOT NULL DEFAULT false,
+    "upload_date" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."province" (
+    "id" SERIAL PRIMARY KEY,
+    "name" text UNIQUE NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."region" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "province_id" int REFERENCES "fertiscan_0.0.9".province(id),
+    "name" text NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."location" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text,
+    "address" text NOT NULL,
+    "region_id" uuid References "fertiscan_0.0.9".region(id)
+    );    
+    
+    CREATE TABLE "fertiscan_0.0.9"."organization_information" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "name" text NOT NULL,
+        "website" text,
+        "phone_number" text,
+        "location_id" uuid REFERENCES "fertiscan_0.0.9".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."organization" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text NOT NULL,
+    "information_id" uuid REFERENCES "fertiscan_0.0.9".organization_information(id),
+    "main_location_id" uuid REFERENCES "fertiscan_0.0.9".location(id)
+    );
+
+
+    Alter table "fertiscan_0.0.9".location ADD "owner_id" uuid REFERENCES "fertiscan_0.0.9".organization(id);
+
+    CREATE TABLE "fertiscan_0.0.9"."sample" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "number" uuid,
+    "collection_date" date,
+    "location" uuid REFERENCES "fertiscan_0.0.9".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."unit" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "unit" text NOT NULL,
+    "to_si_unit" float
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."element_compound" (
+    "id" SERIAL PRIMARY KEY,
+    "number" int NOT NULL,
+    "name_fr" text NOT NULL,
+    "name_en" text NOT NULL,
+    "symbol" text NOT NULL UNIQUE
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."label_information" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "lot_number" text,
+    "npk" text,
+    "registration_number" text,
+    "n" float,
+    "p" float,
+    "k" float,
+    "company_info_id" uuid REFERENCES "fertiscan_0.0.9".organization_information(id),
+    "manufacturer_info_id" uuid REFERENCES "fertiscan_0.0.9".organization_information(id)
+    );
+
+    CREATE TYPE metric_type as ENUM ('volume', 'weight','density');
```

Comment:
I suggest `CREATE TYPE "fertiscan_0.0.9".metric_type as ENUM ('volume', 'weight','density');`.

Same for `LANGUAGE`.

This to avoid having to use the public schema.